src-docs/src/views/filter_group/filter_group_multi.js

```diff
@@ -43,7 +43,20 @@ export default () => {
     { name: 'Robert Schumann' },
     { name: 'Sergej S. Prokofiew' },
     { name: 'Wolfgang Amadeus Mozart' },
-  ];
+  ]);
+
+  function updateItem(index) {
+    if (!items[index]) {
+      return;
+    }
+
+    const newItems = items;
```

Comment:
> However, I noticed that it seems the EuiFilterSelectGroup doesn't re-render until you click off of the pop up containing the EuiFilterSelectItems.

```suggestion
    const newItems = [ ...items ];
```

The component wasn't rerendering because the object reference remained the same. This will change referential equality, ensuring nested object value changes are reflected.

Reply:
Darn, should have known that - thanks for catching that!